### DISCUSSION TITLE
articles/guns-germs-and-steel

### DISCUSSION THREAD

**Discussion by hutusi** [hutusi/hutusi.github.com#142] [Comments]
https://hutusi.com/articles/guns-germs-and-steel 

 美国社会学家贾雷德·戴蒙德的《枪炮、病菌与钢铁》1，源于他在新几内亚岛屿上研究鸟类问题时，他的一位当地朋友问他的一个问题：“为什么白人制造了那么多的货物并将它们运到新几内亚来，而新几内亚黑人却几乎没有属于自己的货物呢？”这个问题又可以展开来这样问：“为什么财富和权力的分配会是现在这个样子，而不是某种别的方式呢？为什么不是印第安人、非洲人和澳大利亚土著杀害、征服或消灭欧洲人或亚洲人呢？” 《枪炮、病菌与钢铁》豆瓣链接: https://book.douban.com/subject/26743265/ ↩

**Comment by tumars** [comment]
买了这本书还一直没看

**Reply by hutusi** [reply]
这本书挺不错，感觉《人类简史》就是借鉴了这本书的观点~
